PTMS/src/App.tsx

```diff
@@ -1,12 +1,23 @@
-import './App.css'
+import { Route, createBrowserRouter,createRoutesFromElements, RouterProvider } from "react-router-dom";
 
-function App() {
+import "./App.css";
+import { Navigation } from "./common/navigation/Navigation";
+
+
+const router = createBrowserRouter(
+  createRoutesFromElements(
+      <Route path="/" element={ <Navigation />}>
+        <Route path="/Home" element={<></>} />
+        <Route path="/CreateProject" element={<></>} />
+        <Route path="/User" element={<></>} />
```

Comment:
bardzo mało istotna uwaga, a w zasadzie pytanie: chcecie mieć URLe z wielkiej litery? raczej się tak nie robi i wygląda to dla mnie dziwnie :smile:

swoją drogą warto sobie jak najwięcej takich "magic stringów" wyciągać do zmiennych i mieć np. `export const paths = { home: '/home', ... }`, potem jak trzeba zmienić ścieżkę to nie trzeba szukać użyć po całej aplikacji, tylko zmieniasz zmienną w jednym miejscu i działa

Reply:
mała a jaka istotna uwaga 😉  poprawiłem na małe litery tak jak powinno być od samego początku! 
przeniosłem paths do osobnego pliku, zgadzam się z Tobą że teraz będzie łatwiej wprowadzać zmiany.